experiment/measurer/test_experiment_coverage_utils.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Google LLC
```

Comment:
I think you should probably write more tests. I'll try to think of some concrete suggestions when I understand this PR better. But on its surface this seems like it won't be easy to maintain without more tests.

Reply:
@BharathMonash We could write tests to check for the following:

- [ ] Check whether functions.csv.gz, segments.csv.gz, and names.csv.gz are empty.
- [ ] Check whether names and IDs are properly resolved (benchmark, fuzzer, function, file, type).
- [ ] Check whether you find the expected values in the right columns for all columns (e.g., afl under fuzzers, file.c:12 file, etc).
- [ ] Check whether there are no duplicate entries in segment_df and function_df after remove_redundant_duplicates is called.
- [ ] Check whether `experiment_specific_df_container` is correctly filled in `measure_all_trials` when there are multiple snapshots (5?) in the multiprocessing pool.

Reply:
Hi @mboehme and @jonathanmetzman, I have added few test cases and changed the name for the file as well. Although, the last suggested test case (with multiprocessing) could not be achieved as python's multiprocessing package has a few OS calls which pyfakefs cannot handle as they don't support for multiprocessing package as of now.

Please let me know if any test case can be improved further or if new test cases can be added.